Lib/pdb.py

```diff
@@ -670,6 +670,19 @@ def set_convenience_variable(self, frame, name, value):
     # Generic completion functions.  Individual complete_foo methods can be
     # assigned below to one of these functions.
 
+    def completenames(self, text, line, begidx, endidx):
```

Comment:
Is this used anywhere?

Reply:
This is overwritting the `completenames` method defined in `cmd.Cmd`, which is called when the user tries to complete the command.

`Pdb` class has some implicit inheritance from `cmd.Cmd` class to achieve customized behaviors. I can add some comments here if it helps.

Reply:
I think adding a test that covers this would be better.

Reply:
I don't think you can (easily) set up a readline test to really trigger the "complete" feature of readline. Even `test_cmd` does not test against complete features for `cmd.Cmd`. I can add some unitttest just to test this method (feeding mock arguments to the function), but a real functional test involving readline would be a large amount of work - and we should probably do that for `cmd` first if we consider adding it.